.devcontainer/postCreateCommand.sh

```diff
@@ -1,5 +1,7 @@
 #! /usr/bin/env bash
-python -m pip install poetry-dynamic-versioning
+# PDV will already have been installed in a devcontainer, but is 
+# re-included here for the benefit of manual dev environments.
+poetry self add "poetry-dynamic-versioning[plugin]"
```

Comment:
This command will not fail for the devcontainer because pdv will already be installed at this point. For users following the manual setup step, whether or not this command succeeds is dependent on whether or not their poetry is installed at the user or root level. I believe it's more common for it to be user-installable which means this should work. 

Regardless, we should not put `sudo` here for two reasons:
1. I think it's not good security practice to put sudo hidden within a script file that you tell a user to run. If something needs elevated privileges you should instead tell the user to run `sudo my-script.sh` instead.
2. Adding `sudo` is likely to cause the command to fail in the more common scenario of a user-installable poetry setup. Frequently in these cases (including for the devcontainer), poetry might be in a root directory, but isn't necessarily added to the `PATH` env variable for the root user. This means `sudo poetry` will fail with a command not found error.

Reply:
Thanks, great thoughts!